packages/calcite-components/src/components/card/card.scss

```diff
@@ -1,87 +1,93 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+ * @prop --calcite-card-background-color: Specifies the background color of the component.
+ * @prop --calcite-card-border-color: Specifies the border color of the component.
+ * @prop --calcite-card-box-shadow: Specifies the box shadow of the component.
+ * @prop --calcite-card-corner-radius: Specifies the corner radius of the component.
+ * @prop --calcite-card-subtitle-color: Specifies the color of the component's `"subtitle"` slot.
```

Comment:
Do we want to provide these tokens for slotted content? We could just set defaults and the user could override themselves on their slotted elements.

Reply:
Places we accept both a prop and a slot, it could make sense to have this. I could also see card having props in the future for title / subtitle, keeping the slot for more advanced cases, so maybe these are fine to leave.